models/DataObject/OwnerAwareFieldInterface.php

```diff
@@ -20,9 +20,16 @@
 interface OwnerAwareFieldInterface
 {
     /**
-     * @param mixed $owner
-     * @param string $fieldname
+     * @param mixed       $owner
+     * @param string      $fieldname
      * @param string|null $language
      */
     public function setOwner($owner, string $fieldname, $language = null);
+
+    /**
+     * @param string $language
+     *
+     * @return void
+     */
+    public function setLanguage(string $language);
```

Comment:
could be dangerous anyway if there is already a setter called "setLanguage". guess we should add it in pimcore 7 but do it in a different way.

but you could  add a setter directly to BlockElement and use it instead of setValue

Reply:
I initially thought about this but abandoned it because it would make the actual language unaware class aware of a property introduced by a trait, that felt unnatural.

But if you like that more I can do it.

Reply:
I would say this is the only option without a BC break

Reply:
Yes, I agree, will do so.

Reply:
thx, could you update the PR accordingly?

Reply:
Ill do tomorrow!